test/controllers/web/sessions_controller_test.rb

```diff
@@ -8,7 +8,7 @@ class Web::SessionsControllerTest < ActionController::TestCase
 
   test 'should post create' do
     password = generate(:string)
-    user = create(:user, { password: password })
+    user = create(:user, { password: password, type: User::USER_TYPES[rand(0..2)] })
```

Comment:
у тебя кода, который проверяет тип пользователя

Reply:
то есть эти изменения, на самом деле, никак не влияют на тест

Reply:
@YJorge 
До добавления этого теста никаких ошибок не было, т.е. подсовывали ему стрингу генерируемую sequence и норм.
А теперь вот такая ошибка падает в этом новом тесте _should_post_create_ 

Error:
Web::SessionsControllerTest#test_should_post_create:
ActiveRecord::SubclassNotFound: The single-table inheritance mechanism failed to locate the subclass: 'string28'. This error is raised because the column 'type' is reserved for storing the class in case of inheritance. Please rename this column if you didn't intend it to be used for storing the inheritance class or overwrite User.inheritance_column to use another column for that information.
    app/forms/session_form.rb:14:in `user'
    app/forms/session_form.rb:20:in `user_valid?'
    app/controllers/web/sessions_controller.rb:9:in `create'
    test/controllers/web/sessions_controller_test.rb:16:in `block in <class:SessionsControllerTest>'

Reply:
в ошибке уже сказано, что происходит не так. полю  type в фабрике пользователя присвоено значение автогенерируемой строки, чего быть не должно. Достаточно убрать это поле из фабрики

Reply:
@YJorge 
Ревертенул эти изменения. Благодарю за ответ